specification/electronic-prescription-service-api.yaml

```diff
@@ -133,7 +133,7 @@ tags:
   - name: dispensing
   - name: prescribing
 paths:
-  /Prepare:
```

Comment:
Github won't let me put a comment there but lines 87 and 125-127 need changing too

Reply:
Removed line 87, updated the other lines